src/content/docs/configuration/v4_migration.md

```diff
@@ -3,132 +3,143 @@ id: v4_migration
+🔗 [AstroUI example configuration](https://github.com/AstroNvim/astroui/tree/main#%EF%B8%8F-configuration)
+📖 `:help astroui`
+The following version 3 `user/init.lua` configuration keys move to the same keys in the version 4 `lua/plugins/astroui.lua` template file:
 
-- The debugging menu has been moved from `<Leader>D` to `<Leader>d` for quicker and more comfortable usage.
+- `colorscheme`
+- `highlights.init`
+- `highlights.<colorscheme>`
+- `icons`
 
-- `H` and `L` have been changed to `[b` and `]b` respectively for changing tabs in the UI. This is for both switching buffers as well as neo-tree sources in the explorer. This can be changed in the your user configuration by adding the following entries to your `mappings.n` table (This uses an internal `astronvim.utils.buffer` function that follows the tab positioning and also allows for using a number to move by multiple tabs at once):
+Additional non version 3 keys configured in the `lua/plugins/astroui.lua` file are:
 
-```lua
-    L = { function() require("astronvim.utils.buffer").nav(vim.v.count > 0 and vim.v.count or 1) end, desc = "Next buffer" },
-    H = { function() require("astronvim.utils.buffer").nav(-(vim.v.count > 0 and vim.v.count or 1)) end, desc = "Previous buffer" },
-```
+- `text_icons` (Same as `icons` except when NerdFont is not installed)
+- `status` (Config for lines and bars built with the `status` API)
 
-- `header` option has been removed in favor of decreasing abstractions. Check the updated [Dashboard Customizations Documentation](../../recipes/alpha)
+### Heirline
 
-- `<Leader>s` has been unified with the `<Leader>f` menu rather than spreading the Telescope mappings out across two menus. Please check the new mappings by pressing `<Leader>f` or in the updated [Mappings Documentation](/mappings)
+Read [Customizing Statusline](/recipes/status) page for how to move the following v3 keys: (?? TODO: do we want to have an heirline.lua file in the template especially since in v3 there were a number of heirline keys??)
 
-- Heirline has moved to a more sustainable configuration format for their `setup` call. Before it was configured with `require("heirline").setup(statusline, winbar, tabline)`, this has moved to a new format with a single table like `require("heirline").setup({ statusline = statusline, winbar = winbar, tabline = tabline, statuscolumn = statuscolumn })`. If you have a custom Heirline configuration please check out the updated [Customizing Statusline Documentation](../../recipes/status) as well as the updated [Heirline Documentation](https://github.com/rebelot/heirline.nvim/blob/master/cookbook.md). (_Note:_ also that along with all of the other core plugin setups, the abstractions have been removed and you will need to update to the new Lazy syntax described in the [Custom Plugins Documentation](../custom_plugins#overriding-core-plugins))
+- `heirline.attributes`
+- `heirline.colors`
+- `heirline.icon_highlights`
+- `heirline.separators`
```

Comment:
These are the `status` keys

Reply:
GitHub won't let me resolve this for some reason. But I've covered this comment off.